doc/multi_group/multi_group_tutorial.rst

```diff
@@ -0,0 +1,111 @@
+
+.. image:: arm1.png
+   :width: 700px
+
+Subsequently add the joints of each arm into their respective groups.
+
+.. image:: arm_joint.png
+   :width: 700px
+
+Moving the two arms in sync
+-----------------------------
+
+First of all define the individual arms and their end-effectors in respective groups as shown -
+
+.. image:: all_groups.png
+   :width: 900px
+
+Now define the end effectors -
+
+.. image:: end_effectors.png
+   :width: 800px
+
+Then combine all the planning groups into a common planning group, in this case - "both_bots" in the srdf file : ::
+
+    <group name="arm1">
+        <joint name="virtual_joint" />
+        <joint name="panda_1_joint_base" />
+        <joint name="panda_1_joint1" />
+        <joint name="panda_1_joint2" />
+        <joint name="panda_1_joint3" />
+        <joint name="panda_1_joint4" />
+        <joint name="panda_1_joint5" />
+        <joint name="panda_1_joint6" />
+        <joint name="panda_1_joint7" />
+        <joint name="panda_1_joint8" />
+        <chain base_link="panda_1_link0" tip_link="panda_1_link8" />
+    </group>
+    <group name="arm2">
+        <joint name="panda_2_joint_base" />
+        <joint name="panda_2_joint1" />
+        <joint name="panda_2_joint2" />
+        <joint name="panda_2_joint3" />
+        <joint name="panda_2_joint4" />
+        <joint name="panda_2_joint5" />
+        <joint name="panda_2_joint6" />
+        <joint name="panda_2_joint7" />
+        <joint name="panda_2_joint8" />
+        <chain base_link="panda_2_link0" tip_link="panda_2_link8" />
+    </group>
+    <group name="robot1">
+        <group name="arm1" />
+        <group name="hand1" />
+    </group>
+    <group name="robot2">
+        <group name="arm2" />
+        <group name="hand2" />
+    </group>
+    <group name="hand1">
+        <joint name="panda_1_hand_joint" />
+        <joint name="panda_1_finger_joint1" />
+    </group>
+    <group name="hand2">
+        <joint name="panda_2_hand_joint" />
+        <joint name="panda_2_finger_joint1" />
+    </group>
+    <group name="both_bots">
+        <group name="arm1" />
+        <group name="arm2" />
+        <group name="hand1" />
+        <group name="hand2" />
+    </group>
+
+We can then define the initial pose of the combined group -
+
+.. image:: home_pose.png
+   :width: 700px
+
+Code for execution
+-------------------------------------------------------
+.. tutorial-formatter:: ./src/multi_group_tutorial.cpp
+
+Moving multiple arms asynchronously
+--------------------------------------
+
+Currently we can not plan for two separate trajectories for two different arms and execute them separately using the ``MoveGroupInterface``.
+The workaround to this is to take the planned trajectories for left and right arm and send them as a goal to the action server of the robot arm .The problem however,
```

Comment:
```suggestion
The workaround to this is to take the planned trajectories for left and right arm and send them as a goal to the action server of the robot arm. The problem however,
```

Reply:
@ur10 No offense, but if you could go through this with a spell checker, that would help a lot.